src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -87,7 +87,7 @@ private Func<
         private bool _tcpKeepAliveEnabled;
 
         private int _maxResponseHeadersLength = HttpHandlerDefaults.DefaultMaxResponseHeadersLength;
-        private int _maxResponseDrainSize = 64 * 1024;
+        private int _maxResponseDrainSize = HttpHandlerDefaults.DefaultMaxResponseDrainSize;
```

Comment:
Is it possible to query the defaults from winhttp and just use those rather than substituting our own?

Reply:
I don't see why not, the API permits it, I just haven't tried it yet.

Reply:
Ok, so I don't think we can query it in any meaningful way here. To query it, you need a handle (even the handler wide session handle should be OK), but we don't have any until we're sending the first request. And we don't want to create the session handle before the first request, because the user still might be tweaking the settings on the handler. We could create another, fake session handle I guess, but  I don't think it's worth.

Reply:
Wouldn't it make sense to add a test that queries `WINHTTP_OPTION_MAX_RESPONSE_DRAIN_SIZE` from a fake handle and checks if our default has the same value? This way we would notice potential future downstream changes.

Reply:
Depends on our intention. If we want to specify (and document) the default value as 1MB independently form WinHttp as implied in https://github.com/dotnet/runtime/pull/109610#pullrequestreview-2423864428, there is little value in any tests.

If we actually mean to stick it to WinHttp defaults, it would validate our definition and would warn us if stuff changes in WinHttp.

No strong opinion here, I'll leve the decision to you, just let's make sure to also open [a doc PR](https://github.com/dotnet/dotnet-api-docs/blob/c5f5dcd0a08005ce43eb893dc38bde3f2bb29370/xml/System.Net.Http/WinHttpHandler.xml#L479-L480)!